vortex-serde/src/file/read/stream.rs

```diff
@@ -79,47 +86,42 @@ impl<R: VortexReadAt> VortexFileArrayStream<R> {
 }
 
 type StreamMessages = Vec<(MessageId, Bytes)>;
-type StreamStateFuture<R> = BoxFuture<'static, VortexResult<(R, StreamMessages)>>;
+type StreamStateFuture = BoxFuture<'static, VortexResult<Vec<(MessageId, Bytes)>>>;
```

Comment:
```suggestion
type StreamStateFuture = BoxFuture<'static, VortexResult<Vec<Message>>>;
```
There's a type called Message to make this slightly nicer